playground/backend/internal/code_processing/code_processing.go

```diff
@@ -189,8 +189,13 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 		return
 	}
 	if !ok {
-		if runOutput.Error != nil {
-			runError.Write([]byte(runOutput.Error.Error()))
+		// If unit test has some error then error output is placed as RunOutput
+		if isUnitTest {
```

Comment:
Shouldn't we add an `else` case to do this?
```go
if runOutput.Error != nil {
	runError.Write([]byte(runOutput.Error.Error()))
```

Reply:
I guess, no. I removed `runOutput.Error` field from `run_output_writer.go`. We do not need this field. This error (`runOutput.Error.Error()`) exists in `errorChannel` and be thrown to the log without using `runError.Write([]byte(runOutput.Error.Error()))`. So we do not need this one:
```
if runOutput.Error != nil {
	runError.Write([]byte(runOutput.Error.Error()))
```
This PR is to receive an error's output that occurs during the processing of the unit tests. This error output is placed in the cache as a `RunOutput` so we need to get this one and keep it to the `runError` value but only if it is a unit tests (`isUnitTest`) and there is an error during processing (`!ok`)